roles/smartgateway/templates/deployment.yaml.j2

```diff
@@ -41,6 +41,9 @@ spec:
       - name: smart-gateway
         image: '{{ metrics_container_image_path }}'
         args:
+{% if use_timestamp %}
```

Comment:
I think you're going to find this fails due to it being undefined. I haven't looked through the entire code yet to check if there is a global default to make sure it's at least defined. Will do that now.

Reply:
Yea so the only use of `use_timestamp` is in the ConfigMap for ceilometer, and the default value is set there inline.

In our case this is probably not super critical since it looks like on Service Telemetry Operator we hard code `useTimestamp: true` in the `SmartGateway` manifest.

I still think we probably just want to declare a default value in `roles/smartgateway/defaults/main.yml` to make this safer.

Reply:
Thanks!